Godeps/Godeps.json

```diff
@@ -1976,6 +1976,7 @@
 		},
 		{
 			"ImportPath": "github.com/inconshreveable/mousetrap",
+			"Comment": "v1.0",
```

Comment:
Interesting that it's adding these back in.

Reply:
🤷‍♂️  who even understands godep?

Reply:
:left-shark: